weblate/trans/views/edit.py

```diff
@@ -893,6 +893,32 @@ def zen(request, path):
 
     search_result, unitdata = get_zen_unitdata(obj, project, unit_set, request)
     sort = get_sort_name(request, obj)
+    if (
+        request.method == "POST"
+        and "merge" not in request.POST
+        and (
+            "accept" in request.POST
+            or "accept_edit" in request.POST
+            or "accept_approve" in request.POST
+            or "delete" in request.POST
+            or "spam" in request.POST
+            or "upvote" in request.POST
+            or "downvote" in request.POST
+        )
```

Comment:
This is essentially a copy of code from `translate` function, it should be shared.

Reply:
Can I just put such a function in the same file, or it should go to some utils.py file?